spec/javascript/components/applications/__snapshots__/ApplicationEditPage.test.js.snap

```diff
@@ -4012,278 +3996,8 @@ Array [
                             onChange={[Function]}
                             onFocus={[Function]}
                             value="a0l0x000000RI8nAAG"
-                          >
-                            <option
-                              value=""
-                            >
-                              Select One...
-                            </option>
-                            <option
-                              value="a0l0P00001JEWXZQA5"
-                            >
-                              Certificate of Preference (COP)
-                            </option>
-                            <option
-                              value="a0l0x000000RI8nAAG"
-                            >
-                              Alice Griffith Housing Development Resident
```

Comment:
These disappearing makes me think that something is going wrong with how preferences are being translated